source_code/relationaldb_examples/basic_db.py

```diff
@@ -1,10 +1,14 @@
 from cdev.resources.simple.relational_db import RelationalDB, db_engine
 
+db_resource = "demo_db"
+username = "username"
+password ="pasword"
+db_name = "default_table"
 
 myDB = RelationalDB(
-  "demo_db",
```

Comment:
It should not use variables directly, but instead use keyword arguments. https://www.w3schools.com/python/gloss_python_function_keyword_arguments.asp

```
myDB = RelationalDB(
  cdev_name="demo_db",
  username="username",
  password="password",
  database_name="default_table"
)
```